test/widgets/new_dm_sheet_test.dart

```diff
@@ -254,17 +254,17 @@ void main() {
   });
 
   group('user selection', () {
-    void checkUserSelected(WidgetTester tester, User user, bool expected) {
-      final icon = tester.widget<Icon>(find.descendant(
-        of: findUserTile(user),
-        matching: find.byType(Icon)));
+      Finder findInUserTile(User user, Finder finder) => find.descendant(of: findUserTile(user), matching: finder);
```

Comment:
This is still wrongly indented. Chris already showed you how this should look; please reread more carefully the comment at https://github.com/zulip/zulip-flutter/pull/1882#discussion_r2548230190 .